text_recognizer/data/iam_synthetic_paragraphs.py

```diff
@@ -16,6 +16,7 @@
 )
 from text_recognizer.data.util import BaseDataset, convert_strings_to_labels, resize_image
 import text_recognizer.metadata.iam_synthetic_paragraphs as metadata
+from text_recognizer.stems.paragraph import ParagraphStem
```

Comment:
Heads up, I [refactored the `stems` a bit](https://github.com/full-stack-deep-learning/fsdl-text-recognizer-2022/tree/main/text_recognizer/stems) a few days ago. Among the reasons to maybe try rebasing this one right before merge.